plugin/completion.py

```diff
@@ -41,10 +42,74 @@
 
 
 class LspResolveDocsCommand(sublime_plugin.TextCommand):
+    def run(self, edit: sublime.Edit, item: dict) -> None:
+        detail = item.get('detail') or ""
+        documentation = minihtml(item.get("documentation"), self.view)
+
+        # don't show the detail in the cooperate AC popup,
+        # if it is alread shown in the AC details filed.
+        self.is_detail_shown = bool(detail)
+
+        content = self.get_content(documentation, detail)
+        self.show_popup(content)
+
+        if not detail or not documentation:
+            # To make sure that the details or documentation fields don't exist
+            # we resove the completion item.
+            # If those fields appear after the item is resolved
+            # update the popup.
```

Comment:
Style nit: No need to be so liberal with line breaks in comments. We have 120 column limit. :)
(Saving vertical space is good).
Also there are some typos in the text.

Reply:
done